client/src/app/shared/directives/resized.directive.ts

```diff
@@ -1,10 +1,15 @@
 import { Directive, ElementRef, Input, OnInit } from '@angular/core';
 
 import { ResizeSensor } from 'css-element-queries';
+import { Interface } from 'readline';
```

Comment:
This import seems wrong

Reply:
jeez how did the linter accept that :rofl: